app/models/user.rb

```diff
@@ -3,5 +3,13 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy, foreign_key: :author_id
   has_many :likes, dependent: :destroy, foreign_key: :author_id
 
-  scope :user_recent_posts, -> { find(1).posts.order(created_at: :desc).first(3) }
+  # scope :user_recent_posts, -> { find(1).posts.order(created_at: :desc).first(1) }
+  scope :user_recent_posts, -> { find(1).posts.order(created_at: :desc).limit(3) }
```

Comment:
- [ ] Kindly remove this comment and this scope (which is incorrect anyways). You already have `recent_post` method. Let's stick to that.